docs/concepts/deposits-withdrawals.md

```diff
@@ -0,0 +1,116 @@
+
+:::note Read more
+[ERC20 bridge logic API documentation](../api/bridge/contracts/ERC20_Bridge_Logic.md#deposit_asset)
+:::
+
+### Diagram: Deposits
+![Deposit diagram](/img/concept-diagrams/diagram-deposit.png)
+
+## Withdrawing assets
+**Assets used for trading** and related activities can only be withdrawn if they're in a general account. That means that, among other things, they can't be in bond for liquidity provision or in a margin account for active orders. 
+
+**VEGA tokens** can only be withdrawn if they are not staked and/or locked.
+
+To remove assets from the Vega network, submit a withdrawal request via a Vega app, such as the Vega Console trading interface, or the API.
+
+This request, if valid, will be put through consensus - the validators sign a multi-signature withdrawal order bundle for the ERC-20 bridge, and assign an expiry. The bridge validates the bundle and then releases the funds to the chosen Ethereum wallet.
+
+If the withdrawal isn't completed before the expiry, the tokens are returned to the participant's collateral account. 
+
+If it's a successful withdrawal transaction, the ERC20 bridge will emit an `Asset_Withdrawn` event, and confirms to the Vega network that the withdrawal has been completed.
+
+:::note Read more
+[ERC20 bridge logic API documentation](../api/bridge/contracts/ERC20_Bridge_Logic.md#withdraw_asset)
+:::
+
+### Withdrawal limits
+Withdrawals can have limits associated with them, where trying to withdraw above a certain amount will cause that withdrawal to be delayed by a set time. If a validator is compromised or otherwise issues a bad withdrawal, the delay gives Vega a chance to stop the withdrawal before it's too late.
+
+The two parts of a withdrawal limit are:
+* **Withdrawal threshold**: Set per asset through governance, requesting to withdraw that amount (and above) will trigger a withdrawal delay
+* **Withdrawal delay**: Set for all assets on the ERC-20 bridge, this is the time that a withdrawal is delayed before it's completed
+
+If you choose an amount to withdraw that is higher than the withdrawal threshold, the multi-signature bundle will only be usable after the withdrawal delay has passed, after which the assets can be moved into an Ethereum wallet.
+
+Once the delay time has passed, and the bundle is valid, **the withdrawal must be completed by the party**, such as by using Vega Console.
+
+:::tip Query for data
+You can see the threshold and delay for withdrawals in [Vega Console ↗](https://console.fairground.wtf).
+
+Alternatively, see the withdraw threshold using a GraphQL query for the asset, and its asset source [ERC20](../graphql/objects/erc20).
+:::
+
+### Diagram: Withdrawals
+![Withdrawal diagram](/img/concept-diagrams/diagram-withdraw.png)
+
+### Withdrawing staked (unlocked) VEGA
+VEGA (an ERC20 token) used for staking is associated with a Vega key. To withdraw unlocked tokens and withdraw them, they must be dissociated first.
```

Comment:
Is this done through the same UI as withdrawals or should we have unstaking docs?

Reply:
It's through the token dapp UI for withdrawals, linked below. I'll also link to the contract function in the "try it" box.